bidding_auction_services_protected_app_signals_egress.md

```diff
@@ -17,71 +17,102 @@ The wire representation of the `egressPayload` will be noised per feature type.
 
 Based on the information in this document, adtechs can write parsers to prepare the `egressPayload` and transform the values it contains into features for use in their model training systems. 
 
-
 # Feature types
 
 We support two kinds of feature types: primitives, which can contain a single feature value; and collections, which can contain multiple primitives.
 
+**Nullability**
+
+Every feature type can optionally be made `nullable`, in which case the value `null` can be represented. By default features are _not_ nullable. A null value for a type is indicated by adding a signaling bit, 0 (for null) or 1 (for non-null), to the least-significant bit of the serialized representation. Consequently, the wire representation for a feature with nullable = true is always 1 bit larger than with nullable = false. 
```

Comment:
Consider quoting "nullable" and "null" with backticks consistently.

Reply:
I reviewed the doc and made one or two minor adjustments for consistency. When I use null and nullable as english words, they are left formatted like the rest of the prose. When used as an example of written code, or used to represent a value in null's case, they are backticked into appearing as code.